### ISSUE TITLE
Open at login notification

### ISSUE THREAD

**Issue by mathguimaraes** [Clipy/Clipy#552]
I've set up the app to run at login, problem is that every time I turn on my mac, I receive macOS notification telling that Clipy was added to open at login. That's the only app that send this notification everyday.

**Comment by falconchen** [comment]
Also troubled by this issue。

**Comment by falconchen** [comment]
I have fix in another way. cancel the startup in clippy.app, and create a plist in `~/Library/LaunchAgents` using launctl startup in macOS.

for me ,it is `~/Library/LaunchAgents/falcon.cmd.clipy.plist`, plist content :

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
  <key>Label</key>
  <string>falcon.cmd.clipy</string>
  <key>ProgramArguments</key>
  <array>
    <string>/Applications/Clipy.app/Contents/MacOS/Clipy</string>
  </array>
  <key>RunAtLoad</key>
  <true/>
  <key>WorkingDirectory</key>
  <string>/Users/falcon</string>
</dict>
</plist>
```

**Comment by will09122000** [comment]
If the above doesn't work for you, like it didn't for me. Try adding the app attached to your login items, it's a simple Automator script to open Clipy when launched.

[ClipyOpen.zip](https://github.com/user-attachments/files/15968163/ClipyOpen.zip)

**Comment by tolluset** [comment]
Same as above, other applications not showing the notification.

**Comment by hugojerez** [comment]
Same here, on every system startup
<img width="335" alt="image" src="https://github.com/user-attachments/assets/21c002ea-7d5c-4c7f-9604-c9bfbdf4787d" />

**Comment by RetroWindow** [comment]
I _finally_ updated to Ventura in the past week and being a big user of login items I immediately ran into the issue with Clipy.

I tried the Automator script option but it failed because the MacOS couldn't scan it for malicious software.

So here it is May 9, 2026 and as far as I can tell this issue has not been successfully addressed by the developer nor is there a workaround that works.  Am I missing something (it wouldn't be the first time ;) ) or should I assume that the problem cannot be fixed one way or the other?

FYI, there is [Autostarter $1.99](https://apps.apple.com/us/app/autostarter/id6444444273?mt=12)  at the Mac App Store which purports to mitigate the login hiding issue for affected apps.  Clipy is the first thing that I tried to fix beforeI figured out that it has a different issue that Autostarter cannot fix. . .  It was only after I did a Kagi.com search that I quickly found out what is going on with Clipy.

**Comment by Econa77** [comment]
Closing this as a duplicate as part of the project restart and issue cleanup.

Please continue the discussion in the remaining issue: https://github.com/Clipy/Clipy/issues/517

For context about the project restart and issue cleanup, please see: https://github.com/Clipy/Clipy/issues/590

This does not mean the report was invalid. I am closing duplicate issues so that future work can be tracked in one place.

If the remaining issue does not cover your case, or if this is still relevant with different details in the current version of Clipy, please feel free to open a new issue with updated information.
